sandbox-nse.tfvars

```diff
@@ -0,0 +1,43 @@
+is_staging                           = false
+is_production                        = false
+terraform_remote_state_bucket_name   = "obi-tfstate-sandbox-nse"
+cell_svc_bucket_name                 = "sbo-cell-svc-perf-test-sandbox-nse"
+ml_paper_bucket_name                 = "ml-paper-bucket-sandbox-hpc-test"
+ml_neuroagent_bucket_name            = "ml-neuroagent-staging"
+nexus_domain_name                    = "sandbox-hpc.openbluebrain.com"
+nexus_obp_bucket_name                = "nexus-obp-sandbox-nse-test"
+nexus_ship_bucket_name               = "nexus-ship-sandbox-nse-test"
+nexus_openscience_bucket_name        = "nexus-openscience-sandbox-nse-test"
+nexus_az_letter_id                   = "a"
+core_web_app_docker_image_url        = "public.ecr.aws/openbraininstitute/core-web-app:staging"
+core_web_app_next_docker_image_url   = "public.ecr.aws/openbraininstitute/core-web-app:entitycore-migration"
+virtual_lab_manager_docker_image_url = "public.ecr.aws/openbraininstitute/virtual-lab-api:20250226.1"
+cell_svc_docker_image_url            = "public.ecr.aws/openbraininstitute/sonata-cell-position:2025.1.1-prod"
+is_nexus_openscience_running         = false
+is_nexus_obp_running                 = true
```

Comment:
Do you need nexus running in NSE sandbox? it not set this to false

Reply:
Nope, set it to false.